packages/cli-repl/src/cli-repl.spec.ts

```diff
@@ -0,0 +1,17 @@
+import mapCliToDriver from './arg-mapper';
+import parseCliArgs from './arg-parser';
+import CliRepl from './cli-repl';
+import { expect } from 'chai';
+
+describe('cli-repl', () => {
+  context('constructor', () => {
+    it('starts without errors', (done) => {
+      const options = parseCliArgs([]);
+      const driverOptions = mapCliToDriver(options);
+      const cli = new CliRepl('mongodb://127.0.0.1:27017', driverOptions, options)
```

Comment:
This will try to connect, we may need to start a server

Reply:
for other readers: this was resolved outside of this PR. We will think of an easier way to test `CliRepl` instances in a different PR.